e2etest/pkg/k8s/nodes.go

```diff
@@ -151,3 +153,62 @@ func conditionStatus(n *corev1.Node, ct corev1.NodeConditionType) corev1.Conditi
 
 	return corev1.ConditionUnknown
 }
+
+func CordonNode(cs kubernetes.Interface, node *corev1.Node) error {
+	if cs == nil {
```

Comment:
I think it's fine to explode here. After all this is a test, we'll have a report saying where it exploded and we have a bit less clutter. Same for node below.